docker/bash/misc.bash

```diff
@@ -0,0 +1,107 @@
+if [ -z "$__bash_lib_misc" ] ; then
+__bash_lib_misc=1
+
+
+source "$BASH_LIB/cdash.bash"
+source "$BASH_LIB/on_exit.bash"
+
+
+__local_config_file="/home/kitware/cdash/app/cdash/config/config.local.php"
```

Comment:
I'm guessing this move to `/home/kitware/cdash/app` is part of a move to have CDash run as non-root?

Reply:
Yes, that is correct. Problems arise from running tests as root, specifically, any files that get created (with default permissions) from the process running as root are not writable by `www-data` which is the default user for Apache. I was actually kind of surprised to find that you can `docker exec -it --user www-data <container> <cmd>` because, as you probably know, it is _not_ possible to `sudo su - www-data` and the like.